browser/ui/views/brave_help_bubble/brave_help_bubble_view.h

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_BROWSER_UI_VIEWS_BRAVE_HELP_BUBBLE_BRAVE_HELP_BUBBLE_VIEW_H_
+#define BRAVE_BROWSER_UI_VIEWS_BRAVE_HELP_BUBBLE_BRAVE_HELP_BUBBLE_VIEW_H_
+
+#include <memory>
+
+#include "base/timer/timer.h"
+#include "brave/browser/ui/views/brave_help_bubble/brave_help_bubble_delegate.h"
+#include "chrome/browser/ui/browser.h"
+#include "ui/views/interaction/element_tracker_views.h"
+#include "ui/views/view.h"
+
+class BraveHelpBubbleView : public views::View,
+                            public BraveHelpBubbleDelegate::Observer,
+                            public views::ViewObserver {
+ public:
+  METADATA_HEADER(BraveHelpBubbleView);
+
+  explicit BraveHelpBubbleView(View* tracked_element);
```

Comment:
If I make this private then I cannot instantiate with std::make_unique - I get a complaint saying the ctor is private. 

The guide here says to avoid using bare new https://chromium.googlesource.com/chromium/src/+/master/docs/ui/learn/bestpractices/ownership.md#lifetime-basics

Reply:
I know you moved c'tor to private section, but FWIW, as you're providing factory function, std::make_unique<> shouldn't be used out of this class. And now you're returning weak ptr of the view, I believe you're absolutely sticking to the rule.